documentation/sphinx/source/release-notes.rst

```diff
@@ -2,7 +2,101 @@
 Release Notes
 #############
 
-7.0.0
+6.2.16
```

Comment:
The release notes need special care when merging. The changes to the release notes on 6.2 should be applied to the 6.2 release notes in the old-release-notes folder, while the 7.0 release notes should be unchanged.

Reply:
For what it's worth, I would prefer that all of our release notes be named with a version so that we didn't have this weird merge quirk.

Reply:
Ah I see - missed that, will take a look

Reply:
Don’t forget to also update the 6.2 release notes in old-release-notes as well

Reply:
And here I was thinking that this will be my most trivial PR :) I think it is correct now.